Gemfile

```diff
@@ -183,4 +183,5 @@ group :development do
   gem 'mailcatcher', '~> 0.6.0'
   gem 'quiet_assets', '~> 1.1.0'
   gem 'launchy', '~> 2.4.0'
+  gem 'spring', '< 2.0.0'
```

Comment:
Where's the version that should be installed? https://github.com/mysociety/alaveteli/blob/067d4ed948539538339b92566b5320782d38b878/Gemfile#L22-L25